Makefile

```diff
@@ -46,6 +47,11 @@ lint:
 	pylint --rcfile .pylint.rc raiden/
 	python setup.py check --restructuredtext --strict
 
+mypy:
+	mypy raiden/transfer/mediated_transfer/target.py --ignore-missing-imports | grep error > mypy-out.txt || true
+# expecting status code 1 from `grep`, which indicates no match
```

Comment:
1. Please indent the comment to the same level as the commands.
2. Can you add a bit more info on that this is just a simple check for block number usage in a single file but that we plan to expand it slowly?

Reply:
I'll do both in this PR.

Reply:
These are done.